angularSTEP/src/app/home/home.component.html

```diff
@@ -1,5 +1,6 @@
 
-<h3 (click)="logout()">Welcome {{name}}!</h3>
+<h3 (click)="logout()">Welcome {{username}}!</h3>
 <div><app-profile-card *ngFor="let user of users | async"
     [user]='user'></app-profile-card>
 </div>
+<app-recipe-card [recipe]="{recipeName:'Brownies', description: 'lhdkjfhkjsdhfkjhdskjhkkkhdsfkjhsdfsfsdfuuhihuhuihhuiuhiuhsffsfsfshuhsuhfishfishfsiu', difficulty: 'Hard'}"></app-recipe-card>
```

Comment:
This isn't probably needed to be merged into main.

Reply:
If you want to check in dummy text until something final is available, I suggest the vernerable lorem ipsum:
https://loremipsum.io/